src/components/workflowEditor/PipelineSelect.tsx

```diff
@@ -1,59 +1,77 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
 import { PipelineSelectProps } from './types'
-import { Modal } from '../common';
-import ci from '../../assets/img/ic-pipeline-ci@2x.png';
-import linkedPipeline from '../../assets/icons/ic-pipeline-linked.svg';
-import webhook from '../../assets/img/webhook.svg';
-
-export class PipelineSelect extends Component<PipelineSelectProps>  {
+import { Modal } from '../common'
+import ci from '../../assets/img/ic-pipeline-ci@2x.png'
+import linkedPipeline from '../../assets/icons/ic-pipeline-linked.svg'
+import webhook from '../../assets/img/webhook.svg'
 
+export class PipelineSelect extends Component<PipelineSelectProps> {
     renderCIMenu() {
-        return <div className="white-card pipeline-webhook" >
-            <div className="white-card__header white-card__header--pipeline-webhook">Select Pipeline</div>
-            <div className="pipeline-webhook__item" onClick={(event) => {
-                this.props.toggleCIMenu(event);
-                this.props.addCIPipeline('CI', this.props.workflowId)
-            }}>
-                <div className="pipeline-webhook__icon">
-                    <img src={ci} className="pipeline-webhook__icon" alt="ci" />
-                </div>
-                <div>
-                    <h4 className="pipeline-webhook__title">Continuous Integration</h4>
-                    <p className="pipeline-webhook__text">Build container image from a source code repository.</p>
+        return (
+            <div className="white-card pipeline-webhook dc__top-radius-4">
+                <div className="dc__window-bg dc__top-radius-4 pt-4 pr-10 pb-4 pl-10 fs-12 fw-6 cn-9">
+                    Build container image
                 </div>
-            </div>
-            <div className="pipeline-webhook__item" onClick={(event) => {
-                this.props.toggleCIMenu(event);
-                this.props.addCIPipeline('LINKED-CI', this.props.workflowId)
-            }}>
-                <div className="pipeline-webhook__icon">
-                    <img src={linkedPipeline} className="pipeline-webhook__icon" alt="linked-ci" />
+                <div
+                    className="flexbox pt-8 pr-12 pb-8 pl-12 pointer"
+                    onClick={(event) => {
+                        this.props.toggleCIMenu(event)
+                        this.props.addCIPipeline('CI', this.props.workflowId)
```

Comment:
Same here. Extract to a different function.

Reply:
done